src/Test/L0/Util/TelemetryPropsUtil.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.TeamFoundation.DistributedTask.WebApi;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+
+namespace Test.L0.Util
+{
+    class TelemetryPropsUtil
+    {
+        public static void AssertPipelineData(Dictionary<string, Object> telemetryProps)
```

Comment:
add params for taking in the values to assert against. default values will be the ones used below